cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -32,7 +33,7 @@ import (
 
 const (
 	KubernetesDirPermissions  = 0700
-	AdminKubeConfigFileName   = "admin.conf"
+	AdminKubeConfigFileName   = "kube-admin.conf"
```

Comment:
Only to align it with the name of the admin defined by `CommonName`. I find it nice that the config has the same name. I can remove.